datasets/norne/README.md

```diff
@@ -0,0 +1,247 @@
+
+- `GPE_LOC`: Geo-political entity, with a locative sense (e.g. "John lives in _Spain_")
+- `GPE_ORG`: Geo-political entity, with an organisation sense (e.g. "_Spain_ declined to meet with Belgium")
+
+The two special types `GPE_LOC` and `GPE_ORG` can easily be altered depending on the task, choosing either the more general `GPE` tag or the more specific `LOC`/`ORG` tags, conflating them with the other annotations of the same type. This means that the following sets of entity types can be derived:
+
+- 7 types, deleting `_GPE`: **`ORG`**, **`LOC`**, `PER`, `PROD`, `EVT`, `DRV`, `MISC`
+- 8 types, deleting `LOC_` and `ORG_`: **`ORG`**, **`LOC`**, **`GPE`**, `PER`, `PROD`, `EVT`, `DRV`, `MISC`
+- 9 types, keeping all types: **`ORG`**, **`LOC`**, **`GPE_LOC`**, **`GPE_ORG`**, `PER`, `PROD`, `EVT`, `DRV`, `MISC`
+
+The class distribution is as follows, broken down across the data splits of the UD version of NDT, and sorted by total counts (i.e. the number of examples, not tokens within the spans of the annotatons):
+
+| Type     | Train  | Dev    | Test   |  Total |
+| :--------|-------:|-------:|-------:|-------:|
+| `PER`    |   4033 |    607 |    560 |   5200 |
+| `ORG`    |   2828 |    400 |    283 |   3511 |
+| `GPE_LOC`|   2132 |    258 |    257 |   2647 |
+| `PROD`   |    671 |    162 |     71 |    904 |
+| `LOC`    |    613 |    109 |    103 |    825 |
+| `GPE_ORG`|    388 |     55 |     50 |    493 |
+| `DRV`    |    519 |     77 |     48 |    644 |
+| `EVT`    |    131 |      9 |      5 |    145 |
+| `MISC`   |      8 |      0 |      0 |      0 |
+
+To access these reduce versions of the dataset, you can use the configs `bokmaal-7`, `nynorsk-7`, `combined-7` for the NER tag set with 7 tags ( **`ORG`**, **`LOC`**, `PER`, `PROD`, `EVT`, `DRV`, `MISC`), and `bokmaal-8`, `nynorsk-8`, `combined-8` for the NER tag set with 8 tags (`LOC_` and `ORG_`: **`ORG`**, **`LOC`**, **`GPE`**, `PER`, `PROD`, `EVT`, `DRV`, `MISC`). By default, the full set (9 tags) will be used.
+
+## Additional Information
+
+### Dataset Curators
+
+NorNE was created as a collaboration between [Schibsted Media Group](https://schibsted.com/), [Språkbanken](https://www.nb.no/forskning/sprakbanken/) at the [National Library of Norway](https://www.nb.no) and the [Language Technology Group](https://www.mn.uio.no/ifi/english/research/groups/ltg/) at the University of Oslo.
+
+NorNE was added to Huggingface Datasets by the AI-Lab at the National Library of Norway.
+
+### Licensing Information
+
+The NorNE corpus is published under the same [license](https://github.com/ltgoslo/norne/blob/master/LICENSE_NDT.txt) as the Norwegian Dependency Treebank
+
+### Citation Information
+
+This dataset is described in the paper _NorNE: Annotating Named Entities for Norwegian_ by
+Fredrik Jørgensen, Tobias Aasmoe, Anne-Stine Ruud Husevåg, Lilja Øvrelid, and Erik Velldal, accepted for LREC 2020 and available as pre-print here: https://arxiv.org/abs/1911.12146.
```

Comment:
````suggestion
Fredrik Jørgensen, Tobias Aasmoe, Anne-Stine Ruud Husevåg, Lilja Øvrelid, and Erik Velldal, accepted for LREC 2020 and available as pre-print here: https://arxiv.org/abs/1911.12146.

```

@inproceedings{johansen2019ner,
  title={NorNE: Annotating Named Entities for Norwegian},
  author={Fredrik Jørgensen, Tobias Aasmoe, Anne-Stine Ruud Husevåg,
          Lilja Øvrelid, and Erik Velldal},
  booktitle={LREC 2020},
  year={2020},
  url={https://arxiv.org/abs/1911.12146}
}
```
````
you can add the bibtex citation here

Reply:
Also can you add this at the end of the dataset card ? :)

```

### Contributions

Thanks to [@versae](https://github.com/versae) for adding this dataset.
```

Reply:
Done